android/assets/jsons/translations/Portuguese.properties

```diff
@@ -653,27 +653,27 @@ Bombard = Bombardear
 NUKE = Usar bomba nuclear
 Captured! = Capturado(a)!
  # Requires translation!
-defence vs ranged = 
+defence vs ranged = defesa contra ataques a distância
 [percentage] to unit defence = [percentage] a defesa da unidade
  # Requires translation!
-Attacker Bonus = 
+Attacker Bonus = Bonus de Atacante
  # Requires translation!
-Landing = 
+Landing = Desembarcando
  # Requires translation!
-Flanking = 
+Flanking = Flanqueando
  # Requires translation!
-vs [unitType] = 
+vs [unitType] = vs [unitType]
  # Requires translation!
-Terrain = 
+Terrain = Terreno
 
 
 Hurry Research = Apressar Pesquisa
 Conduct Trade Mission = Realizar uma missão comercial
  # Requires translation!
-Your trade mission to [civName] has earned you [goldAmount] gold and [influenceAmount] influence! = 
+Your trade mission to [civName] has earned you [goldAmount] gold and [influenceAmount] influence! = Sua missão comenrcial ao(à) [civName] lhe rendeu [goldAmount] de ouro e [influenceAmount] de influência
```

Comment:
comercial?

Reply:
> comercial?

Yes, it should be "comercial".
Can I change this directly in this answer? It doesn't seems to be editable.